### ISSUE TITLE
Line numbers using info color as background makes them impossible to read when identical

### ISSUE THREAD

**Issue by Rassi** [lauriro/json-lite#56]
Firefox 82.0.1 (64-bit) on Windows 10

I use custom colors, and have set both `info color` and `info hover color` to `#333`. This results in line number background being the same as the text color making it impossible to read:

![image](https://user-images.githubusercontent.com/3517495/97843740-1d093600-1cea-11eb-9216-c9abee642de0.png)

**Comment by lauriro** [comment]
That's the power of custom colors :)

As you figured out, current implementation reuses `info` and `info hover` colors for line numbers and line numbers background.
With custom colors it is up to you to find the combination you like most.

There are already many options to configure

- you can use different color for `info` and `info hover`
- you can add white shadow around text, eg. set `info hover` to `#333;text-shadow: 0px 0px 6px #fff`
- you can turn off line numbers
- you can use default theme

I am not sure, would more options do any good.

**Comment by Rassi** [comment]
Ok, I see that I might have missed out explaining why I use those settings.

I prefer my text views to not change on hover, and drop shadows are also something that I prefer not to have in "plain text". This is why I figured out that I could remove the drop shadow style, and set the hover and info colors to be identical, which gives me what I would like for "static/normal plain text". I then updated Firefox and your extension today, and wondered what the black margin was, until I figured out it was due to line numbers being added. 

I might want to try out line numbers, but it is completely fine if you do not want to add more options. I will just disable line numbers then.

Thanks for an otherwise great extension :)
